src/test/java/de/tum/in/www1/artemis/programmingexercise/ProgrammingExerciseTestService.java

```diff
@@ -738,6 +741,72 @@ public void testImportProgrammingExercise_scaChange_activated() throws Exception
         assertThat(exerciseToBeImported.getMaxStaticCodeAnalysisPenalty()).isEqualTo(80);
     }
 
+    /**
+     * Method to test the correct import of a programming exercise into an exam during an exam import
+     * For more Information see {@link de.tum.in.www1.artemis.service.exam.ExamImportService}
+     */
+    public void importProgrammingExerciseasPartOfExamImport() throws Exception {
```

Comment:
```suggestion
    public void importProgrammingExerciseAsPartOfExamImport() throws Exception {
```
Not that I would really care, but just saw it.